jetty-server/src/main/config/modules/ssl-reload.mod

```diff
@@ -0,0 +1,21 @@
+DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
```

Comment:
This is a comment.
Add `#` to the start please.

Reply:
None of the other module files have this as a comment? I don't think this is supposed to have a `#`.

Reply:
The other modules are in the wrong.

Put this on your other PR for cleanup.

```
$ git grep "# DO NOT EDIT"
jetty-cdi/src/main/config/modules/cdi-decorate.mod:# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
jetty-cdi/src/main/config/modules/cdi-spi.mod:# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
jetty-cdi/src/main/config/modules/cdi.mod:# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
jetty-cdi/src/main/config/modules/cdi2.mod:# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
jetty-deploy/src/main/config/modules/decorate.mod:# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
jetty-home/src/main/resources/modules/setuid.mod:# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
jetty-server/src/main/config/modules/https.mod:# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
```